tests/test_data/rendered_templates/default.rst

```diff
@@ -52,9 +53,10 @@ Schemes
 ^^^^^^^
 
 .. csv-table::
-   :header: ID,Dtype,Min,Labels,Mappings
+    :header-rows: 1
```

Comment:
Nice. Using the `:header-rows: 1` also helps writing neater templates.

Reply:
Yes, I also found this the easier solution.